src/controllers/hotkeys/ActionNames.hpp

```diff
@@ -89,6 +89,7 @@ inline const std::map<HotkeyCategory, ActionDefinitionMap> actionNames{
               1,
           }},
          {"scrollToBottom", ActionDefinition{"Scroll to the bottom"}},
+         {"scrollToTop", ActionDefinition{"Scroll to the top"}},
```

Comment:
These actions are sorted in alphabetical order by display name. Maybe it would be best to change the display names of these actions to have them the other way around in the UI.